File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        int answer = 0;
        // price >= 100000 // 5% 
        // price >= 300000 // 10%
        // price >= 500000 // 20%
        double discountRate = price >= 500000 ? 0.8 : price >= 300000 ? 0.9 : price >= 100000 ? 0.95 : 1;
        answer = (int)(price * discountRate);
        return answer;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    # 10만 원 이상 사면 5%
    # 30만 원 이상 사면 10%
    # 50만 원 이상 사면 20%를 할인
    if price >= 500000:
        return (price * 0.8) // 1
    elif price >= 300000:
        return (price * 0.9) // 1
    elif price >= 100000:
        return (price * 0.95) // 1
    else:
        return price
    